plugins/modules/launchdarkly_project_copy.py

```diff
@@ -12,29 +12,29 @@
 DOCUMENTATION = r"""
 ---
 module: launchdarkly_project_copy
-short_description: Copy source LaunchDarkly Project in or between accounts.
+short_description: Copy a project
 description:
-     - Copy source LaunchDarkly Project in or between accounts. This can be used to make a clone of an existing project.
+     - Copy a LaunchDarkly project within or between accounts. You can use this to make a clone of an existing project, or to copy data from one account to another.
```

Comment:
Do we want to add a call out, not all aspects may be copied or something more specific in general? This was accurate as of the original time of writing, but no longer is. Such as `workflows` and `triggers` are not copied. Maybe we just be explicit what is copied?

Reply:
Added a link to our guide that talks about this in more detail.